src/utils.tsx

```diff
@@ -61,3 +65,19 @@ export const getPathFromQueryString = (history: History) => {
 
   return typeof path === 'string' && path.length ? path : null;
 };
+
+type GetCodeLineAnchorGetterParams = {
+  _getCodeLineAnchor?: typeof getCodeLineAnchor;
+  compareInfo?: CompareInfo | null | void;
```

Comment:
It would be a mistake if the caller *omitted* this parameter and doing so would trigger incorrect behavior (without triggering any type errors). I suggest making this non-optional so that the caller is forced to do this:

```tsx
getCodeLineAnchorGetter({
  // This makes it clear that passing an undefined object was intentional.
  compareInfo: undefined
});
```

Reply:
Good point. This has been resolved.